### PR TITLE
Musl bindings

### PR DESCRIPTION
**Pull request by richarddd**
Fixes: https://github.com/DelSkayn/rquickjs/issues/25

### CODE REVIEW

**Comment by katyo** [comment]
@richarddd Could you rebase this PR on current master?

**Comment by richarddd** [comment]
> @richarddd Could you rebase this PR on current master?

Will do! 👌

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -287,7 +287,6 @@ jobs:
 
           echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_RUNNER=qemu-aarch64" >> $GITHUB_ENV
           echo "/musl/bin" >> $GITHUB_PATH
-
```
Comment (katyo) [inline .github/workflows/ci.yml]: Remove this commit please.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -63,13 +63,12 @@ jobs:
...
             target/doc/rquickjs-sys
             target/doc/rquickjs-macro
-
   check:
     runs-on: ubuntu-latest
     permissions:
       pull-requests: write
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
```
Comment (katyo) [inline .github/workflows/ci.yml:71]: Seems I forget to update this. Thanks!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (richarddd) [reply .github/workflows/ci.yml]: ? It’s needed for musl builds for arm on x86

**Pull request merged**